### PR TITLE
Updated the instruction with env file

### PR DESCRIPTION
**Pull request by wolfdancer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,16 +13,16 @@ git clone https://github.com/rackerlabs/docker-graphite-api-blueflood-finder.git
 cd docker-graphite-api-blueflood-finder/
 ```
 
-# Run graphite-api service with blueflood as datasource
+# Create enviroment variables file
+
+Environment variables file is recommneded especially in the case you need to specify API key.
```
Comment (ChandraAddala) [inline README.md]: Typo here "recommneded"

**Comment by ChandraAddala** [comment]
Other than the minor comment, looks good.

**Comment by wolfdancer** [comment]
Thanks!  Ran the spell check with the content this time.

**Comment by ChandraAddala** [comment]
Looks good.

**Pull request merged**